test/serializers/attributes_test.rb

```diff
@@ -23,6 +23,11 @@ def test_attributes_with_fields_option
           @profile_serializer.attributes(fields: [:name]))
       end
 
+      def test_attributes_with_fields_method
```

Comment:
No `fields` method case could be more explicitly tested. There is already a case that covers that behaviour, though (https://github.com/lautis/active_model_serializers/blob/fields-filtering/test/serializers/attributes_test.rb#L39).